src/base/error_code.h

```diff
@@ -1703,8 +1703,9 @@
 #define ER_FLASHBACK_EXCEED_MAX_NUM_TRAN_TO_SUMMARY -1335
 #define ER_FLASHBACK_SCHEMA_CHANGED                 -1336
 #define ER_FLASHBACK_LOG_NOT_EXIST                  -1337
+#define ER_FLASHBACK_EXCEED_THRESHOLD               -1338
```

Comment:
How about `ER_FLASHBACK_TIMEOUT` or something? This name doesn't contain the meaning of what kind of THRESHOLD it is.

Reply:
Yes, I will change it to something. 
this error occurs when timeout between requests, so it is better to be replaced with `ER_FLASHBACK_TIMEOUT`